atala-prism-sdk/src/commonMain/kotlin/io/iohk/atala/prism/walletsdk/prismagent/PrismAgent.kt

```diff
@@ -990,6 +1031,128 @@ class PrismAgent {
         )
     }
 
+    suspend fun initiatePresentationRequest(
+        type: CredentialType,
+        toDID: DID,
+        proofTypes: Array<ProofTypes>,
+        withResolver: DIDResolver
```

Comment:
This is not needed. The `PrismAgent` has a `Castor` member, that has a `resolveDID()` method.